pages/styles/events.css

```diff
@@ -36,8 +16,14 @@
   flex-direction: column;
 }
 
-.meetupCard div > div:first-child {
-  display: none;
+.meetupCard > div > div:nth-child(2) {
+  margin-bottom: -3.6em;
+  text-align: center;
+}
+
+.meetupCard > div > div:nth-child(2) a:hover {
```

Comment:
Can you define these rules within the context of the FlatCard?

They'll always need to be this way. Not just on this page.


```
{Button && <div className={classNames(styles.flatCardButton, styles.alignCenter)}>{Button}</div>}

//////

.flatCardButton > button:focus,
.flatCardButton > button:hover,
.flatCardButton > a:focus,
.flatCardButton > a:hover {
  background-color: var(--white);
  color: var(--secondary);
}
```